scripting/get5/eventlogger.sp

```diff
@@ -61,6 +61,14 @@ static void AddPlayer(JSON_Object params, const char[] key, int client) {
   params.SetString(key, value);
 }
 
+static void AddIpAddress(JSON_Object params, int client) {
+    char value[15];
```

Comment:
Let's go ahead and just use a 32-char buffer so it works with IPV6 (for the future) as well.